index.js

```diff
@@ -156,14 +159,16 @@ function customRenderFn(renderFn) {
 /**
  * Render a string using a template engine
  *
+ * @this {webpack.loader.LoaderContext}
  * @param {any} engine
  * @param {string} str
  * @param {Object} locals
  * @returns {string}
  */
 function render(engine, str, locals, engineOptions) {
+  let output = null;
   try {
-    var output = engine.render(engine.engine, str, locals, engineOptions)
+    output = engine.render.call(this, engine.engine, str, locals, engineOptions)
```

Comment:
Add a comment here about the identity of `this` in this context.

Reply:
https://github.com/jabney/render-template-loader/pull/6/files#diff-168726dbe96b3ce427e7fedce31bb0bcR162 is not enough ?

Reply:
That works!